tests/unit_tests/test_image_series.py

```diff
@@ -125,3 +127,26 @@ def test_check_image_series_external_file_relative(self):
 def test_check_image_series_external_file_valid_pass_non_external():
     image_series = ImageSeries(name="TestImageSeries", rate=1.0, data=np.zeros(shape=(3, 3, 3, 3)), unit="TestUnit")
     assert check_image_series_external_file_valid(image_series=image_series) is None
+
+
+def test_check_large_image_series_stored_internally():
```

Comment:
We usually right two tests - one that raises a message and another that does not.

Can you write another that does not (that is, shows the 'correct' behavior)?

Reply:
All right, done.